deploy/modules/aws/tidb-cluster/local.tf

```diff
@@ -76,14 +76,15 @@ locals {
       # additional_userdata  = file("userdata.sh")
     },
     {
-      name                 = "${var.cluster_name}-tidb"
-      key_name             = var.ssh_key_name
-      instance_type        = var.tidb_instance_type
-      root_volume_type     = "gp2"
-      root_volume_size     = "50"
-      public_ip            = false
-      kubelet_extra_args   = join(" ",
+      name             = "${var.cluster_name}-tidb"
+      key_name         = var.ssh_key_name
+      instance_type    = var.tidb_instance_type
+      root_volume_type = "gp2"
+      root_volume_size = "50"
+      public_ip        = false
+      kubelet_extra_args = join(" ",
         [
+          "--allowed-unsafe-sysctls=\\\"net.*\\\"",
```

Comment:
is PodSecurityPolicy required? https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#podsecuritypolicy

Reply:
No, by default all the sysctls are allowed by `PodSecurityPolicy`